admin/notif-services/jira.rst

```diff
@@ -13,7 +13,7 @@ To send Observability Cloud alert notifications to Jira, perform the following c
 
 * :ref:`jira1`
 
-  You must be an Observability Cloud administrator to perform this task.
+ NOTE: You must be an Observability Cloud administrator to perform this task.
```

Comment:
Instead of using "NOTE:", we use an admonition directive. You can see an example of this directive at the beginning of the [Create and manage organization access tokens using Splunk Observability Cloud](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html) topic.

The code for this admonition is in the `admin/authentication-tokens/org-tokens.rst` file in the repo. The line starts with `.. note:: By default, only users who are administrators can search for and view all access tokens.`

Even so, I'm also confused by the text! The note is in the middle of two steps. That's a mistake. By reading more, I see that the note applies to the first step. That means that the note needs to be _before_ the first step.

The fix is to replace the line with `.. note: You must be an Observability Cloud administrator to perform this step.`